packages/web/src/scss/components/Dropdown/_Dropdown.scss

```diff
@@ -1,14 +1,50 @@
+@use 'sass:map';
+@use '../../tools/breakpoint';
 @use 'theme' as theme;
 
 .Dropdown {
+    position: absolute;
+    z-index: 1;
     display: none;
     width: max-content;
+    max-width: 100%;
     padding: theme.$padding;
     border-radius: theme.$border-radius;
     background-color: theme.$background;
     box-shadow: theme.$shadow;
+
+    @each $breakpoint-name, $breakpoint-value in theme.$breakpoints {
+        &[data-breakpoint='#{$breakpoint-name}'] {
+            @include breakpoint.up(map.get(theme.$breakpoints, $breakpoint-name)) {
+                width: max-content;
```

Comment:
Unfortunately, this CSS gets never applied. Please try:

1. Moving the loop after current lines 25–28.
2. If 1) doesn't work, add an `!important` here.